deploy/test/test_cases/TEST_ID_10_SECRETS_DESTINATION_not_exist.sh

```diff
@@ -9,6 +9,7 @@ echo "Create test-env pod. SECRETS_DESTINATION is with invalid value 'incorrect_
 export SECRETS_DESTINATION_KEY_VALUE=$KEY_VALUE_NOT_EXIST
 deploy_init_env
 
+cli_get_pods_test_env
```

Comment:
i understand we have some failures because of this, but we need to find better way.